Source/Scene/Scene.js

```diff
@@ -3358,6 +3432,118 @@ define([
         return result;
     }
 
+    var writeLogDepthRegex = /\s+czm_writeLogDepth\(/;
+    var vertexlogDepthRegex = /\s+czm_vertexLogDepth\(/;
+    var extensionRegex = /\s*#extension\s+GL_EXT_frag_depth\s*:\s*enable/;
+
+    function getLogDepthShaderProgram(context, shaderProgram) {
```

Comment:
This is a lot of new code in Scene.js.  Perhaps move these two new log depth shader functions to a new private utility class in a separate file?